exercises/concept/treasure-chest/tests/Tests.elm

```diff
@@ -0,0 +1,33 @@
+module Tests exposing (tests)
+
+import Expect
+import Test exposing (..)
+import TreasureChest exposing (..)
+
+
+tests : Test
+tests =
+    describe "TreasureChest"
+        [ describe "1"
```

Comment:
The `"1"` tests should pass when the conditions are cleared for the first task, but currently they do not, because the code won't compile without `getTreasure` defined.
I suggest to add it naked, without a signature or arguments:
```elm
getTreasure =
  Debug.todo """
define getTreasure, with two parameters:
- a String (for the passwordAttempt)
- a TreasureChest custom type
 and returns:
- if the passwordAttempt does not match the String in the TreasureChest
  - Nothing
- if the passwordAttempt matches the String in the TreasureChest
  - Just treasure (where treasure is the generic in the TreasureChest)
"""
```
(but either simplify the text or make it match exactly the instructions once they are finalized)